code/game/objects/items/weapons/dna_injector.dm

```diff
@@ -1190,3 +1190,35 @@
 	New()
 		block = REMOTETALKBLOCK
 		..()
+
+/obj/item/weapon/dnainjector/nofail/randompower/New(newloc)
+    var/type = pick(/obj/item/weapon/dnainjector/nofail/hulkmut,
```

Comment:
>checking every place the injectors are referenced and making sure they still behave as intended

Why wouldn't they? `pick(existing_typesof(/obj/item/weapon/dnainjector/nofail))` just returns a random child of `/obj/item/weapon/dnainjector/nofail`, it doesn't touch the functionality of injectors at all.